grants_tagger_light/training/cli_args/train_args.py

```diff
@@ -30,7 +30,7 @@ class BertMeshTrainingArguments(TrainingArguments):
         default=8
     )  # set to 256 in grants-tagger repo
     per_device_eval_batch_size: int = field(default=8)
-    gradient_accumulation_steps: int = field(default=1)
+    gradient_accumulation_steps: int = field(default=2)
```

Comment:
we should not change the defaults ideally, just the params that get passed

Reply:
put back the default to 1 👍